client/components/Editor/styles/figure.scss

```diff
@@ -1,29 +1,43 @@
 figure {
+	display: block;
```

Comment:
IIRC these changes don't result in any visible differences, it just makes the figure boundaries conveniently wider, right? If so — please check on what happens in the full-bleed (breakout) mode if you haven't already 🙂